src/main/java/com/ivanmostovyi/demo/controller/MyErrorController.java

```diff
@@ -0,0 +1,33 @@
+package com.ivanmostovyi.demo.controller;
+
+import org.springframework.boot.web.servlet.error.ErrorController;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.Objects;
+
+@Controller
+public class MyErrorController implements ErrorController {
+
+    @GetMapping("/error")
+    public String handleError(Model model, HttpServletRequest request){
+
+        Integer errorCode = (Integer) request.getAttribute("javax.servlet.error.status_code");
+        Exception exception = (Exception) request.getAttribute("javax.servlet.error.exception");
+
+        model.addAttribute("error_number", errorCode);
+
+        if (exception != null){
+            model.addAttribute("exception", exception.getMessage());
```

Comment:
`errorMessage` would be better suitable name.

I'll just refer [this comment](https://github.com/PNUdev/pnu-feedback/pull/22#issuecomment-748506373) not to retype similar text again.
Naming is pretty valuable.